src/codeu/chat/util/TokenizerTest.java

```diff
@@ -0,0 +1,25 @@
+package codeu.chat.util;
+
+import java.io.IOException;
+import static org.junit.Assert.*;
+import org.junit.Test;
+
+public final class TokenizerTest {
+  @Test
+  public void testWithQuotes() throws IOException {
+    final Tokenizer tokenizer = new Tokenizer(“hello world how are you”);
+    assertEquals(tokenizer.next(), “hello”);
```

Comment:
Seems like the quotes were just copied from the instructions, you should retype the quotes yourself since they are not characters that are interpreted by the compiler.